packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/ListView/index.test.js

```diff
@@ -219,6 +276,7 @@ describe('<ListView>', () => {
       });
 
       it('uses the root HTML element to determine if the view has changed', () => {
+        expect(instance).toBeDefined();
```

Comment:
for the future, this seems redundant. If it's not defined the test will still fail on null ptr dereference in the next line.